src/components/dialogs/parameters/common/limitreductions/columns-definitions.ts

```diff
@@ -84,6 +91,23 @@ export const getLimitReductionsFormSchema = (nbTemporaryLimits: number) => {
                     ...getLimitDurationsFormSchema(nbTemporaryLimits),
                 })
             ),
+            [PARAM_SA_FLOW_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_LOW_VOLTAGE_ABSOLUTE_THRESHOLD]: yup.number().required(),
+            [PARAM_SA_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD]: yup
+                .number()
+                .min(0, 'NormalizedPercentage')
+                .max(100, 'NormalizedPercentage')
+                .required(),
+            [PARAM_SA_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD]: yup.number().required(),
```

Comment:
This is shared between LF parameters and SA parameters, I don't think we want checks specific to SA here.

Reply:
fixedi removed this change ad added specific schema rules to AS